saltstack/base/salt/prerequisites/usr/bin/user-data-helper.sh

```diff
@@ -1,6 +1,19 @@
 #!/bin/bash
 
 set -ex
+
+ACC_KEY_TMP_FILE=$(mktemp)
```

Comment:
Nope, this should reside in a location that is already in the LUKS volume. `mktemp` will likely place in `/tmp`. This is a sensitive file, we do not want it to appear on the root disk unprotected.

I suggest using a location under `/etc/ccmv2/` using some fixed file name.